bigchaindb/__init__.py

```diff
@@ -59,6 +59,10 @@
         'workers': None,  # if none, the value will be cpu_count * 2 + 1
         'threads': None,  # if none, the value will be cpu_count * 2 + 1
     },
+    'wsserver': {
+        'host': os.environ.get('BIGCHAINDB_WSSERVER_HOST') or 'localhost',
```

Comment:
I noticed that the `ws` prefix is used quite a lot to name libraries and variables. I was thinking if using `websocket` or `wsserver`. At the end I thought that `websocket` is the name of a protocol, while `wsserver` is more aligned with the configuration key `server`. I'd like to rename `server` to `apiserver` actually :sweat_smile:.